CHANGELOG.md

```diff
@@ -4,6 +4,33 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v13.4.1](https://github.com/voxpupuli/puppet-prometheus/tree/v13.4.1) (2024-03-25)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-prometheus/compare/v13.4.0...v13.4.1)
+
+**Breaking changes:**
```

Comment:
There are breaking changes in the release, so please make this a 14.0.0 release

Reply:
Done, however the PR title is the same. Cheers!